src/adagrams.js

```diff
@@ -1,6 +1,110 @@
+import { ENGINE_METHOD_DIGESTS } from "constants";
+
+const hashIntoObject = arrOfLetters => {
+  let object = {};
+  arrOfLetters.forEach(letter => {
+    object[letter] ? object[letter] += 1 : object[letter] = 1;
```

Comment:
Could you come up with a more descriptive name for this function and its local variables? Maybe

```js
const letterFrequencies = arrOfLetters => {
  const frequencies = {};
  // ...
  return frequencies;
}
```